### PR TITLE
Reduce TTL for DNS

### PR DESCRIPTION
**Pull request by thompiler**
This TTL was causing downtime during the switch over when testing the new notice-ingress pod configuration. Howie and Brandon found that reducing this value reduces the downtime from ~3 min to 3 seconds.

Context in Slack: https://logicmonitor.slack.com/archives/C043ETM5VRS/p1664478987483429?thread_ts=1664475996.186739&cid=C043ETM5VRS

### Applying the change
To apply this change in EKS you must run this for staging and production:

```
awsume airbrake-production
```

```
helm upgrade -i appmesh-controller appmesh-controller \
  --namespace appmesh-system \
  --set region=us-east-1 \
  --set serviceAccount.create=false \
  --set serviceAccount.name=appmesh-controller \
  --kubeconfig ~/.kube/eks-production
```

### CODE REVIEW

**Comment by Hoggles** [comment]
Do we think that 3s is a good amount of time? Should we go down to 1s?

**Comment by thompiler** [comment]
>
Do we think that 3s is a good amount of time? Should we go down to 1s?


Lowest we can do is probably best if there are no big side effects.

👍

File: stable/appmesh-controller/values.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ cloudMapCustomHealthCheck:
 
 cloudMapDNS:
   # cloudMapDNS.ttl if set will use this global ttl value
-  ttl: 300
+  ttl: 3
```
Comment (Hoggles) [inline stable/appmesh-controller/values.yaml:73]:
```suggestion
  ttl: 30
```

Comment (Hoggles) [reply stable/appmesh-controller/values.yaml:73]: After the change to goab, going up to 30s

File: stable/appmesh-controller/values.yaml [hunk 2]
```diff
@@ -70,7 +70,7 @@ cloudMapCustomHealthCheck:
 
 cloudMapDNS:
   # cloudMapDNS.ttl if set will use this global ttl value
-  ttl: 300
+  ttl: 30
```
Comment (Hoggles) [inline stable/appmesh-controller/values.yaml]:
```suggestion
  ttl: 3
```

**Pull request merged**